qtpy/tests/test_main.py

```diff
@@ -23,6 +28,16 @@ def assert_pyside2():
     assert QtWidgets.QWidget is PySide2.QtWidgets.QWidget
     assert QtWebEngineWidgets.QWebEnginePage is PySide2.QtWebEngineWidgets.QWebEnginePage
 
+def assert_pyside6():
+    """
+    Make sure that we are using PySide
+    """
+    import PySide6
+    assert QtCore.QEvent is PySide6.QtCore.QEvent
+    assert QtGui.QPainter is PySide6.QtGui.QPainter
+    assert QtWidgets.QWidget is PySide6.QtWidgets.QWidget
+    #assert QtWebEngineWidgets.QWebEnginePage is PySide6.QtWebEngineWidgets.QWebEnginePage
```

Comment:
Maybe we should add a comment here describing that only after PySide6 6.2 support for QtWebEngineWidgets will be added